pkgs/tools/admin/cli53/default.nix

```diff
@@ -1,30 +1,19 @@
-{ lib, python2, fetchurl }:
+# This file was generated by https://github.com/kamilchm/go2nix v2.0-dev
+{ lib, stdenv, buildGoPackage, fetchgit }:
 
-python2.pkgs.buildPythonApplication rec {
+buildGoPackage rec {
   name = "cli53-${version}";
-  version = "0.4.4";
+  version = "0.8.12";
 
-  src = fetchurl {
-    url = "mirror://pypi/c/cli53/${name}.tar.gz";
-    sha256 = "0s9jzigq6a16m2c3qklssx2lz16cf13g5zh80vh24kxazaxqzbig";
-  };
-
-  postPatch = ''
-    substituteInPlace setup.py --replace "'argparse', " ""
-  '';
+  goPackagePath = "github.com/barnybug/cli53";
 
-  checkPhase = ''
-    ${python2.interpreter} -m unittest discover -s tests
-  '';
-
-  # Tests do not function
-  doCheck = false;
+  src = fetchgit {
+    rev = "2624c7c4b38a33cdbd166dad1d3e512830f453e4";
+    url = "https://github.com/barnybug/cli53.git";
+    sha256 = "0bhczmzrgf7ypnhhzdrgnvg8cw8ch1x1d0cgajc5kklq9ixv9ygi";
```

Comment:
Since there seems to be no git submodules in this project, `fetchFromGitHub` is the preferred method.